.travis.yml

```diff
@@ -1,16 +1,25 @@
 ---
+dist: trusty
+sudo: false
+
 cache:
   directories:
     - "$(npm config get prefix)/bin"
 
 language: python
 python: '2.7'
 
+addons:
+  apt:
```

Comment:
According to [this](https://github.com/koalaman/shellcheck, it looks like we need the following in order to make sure shellcheck is in our package lists.
```
sources:
    - debian-sid
```